core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -396,7 +398,12 @@ angular.module('oppia').directive('storyNodeEditor', [
             $scope.chapterOutlineButtonsAreShown = (
               !$scope.chapterOutlineButtonsAreShown);
           };
-
+          $scope.addFocusWithoutScroll = function(label) {
+            FocusManagerService.setFocus(label);
+            setTimeout(function() {
+              window.scrollTo(0, 0);
+            }, 5);
```

Comment:
Ditto: auto scroll

Reply:
It is prevented so that the user doesn't miss seeing the TO-DO list in small screens.